src/components/Header.js

```diff
@@ -8,7 +8,12 @@ import headerBackgroundSvg from '../assets/header-background.svg'
 import logoSvg from '../assets/logotext.svg'
 
 function Header({ compact }) {
-  const BeeIcon = <img src={beeSvg} height={compact ? 40 : 60} alt="" />
+  const BeeIcon = (
+    <a href="https://1hive.org">
```

Comment:
We can use the [`Link`](https://ui.aragon.org/link/) component. from `@1hive-ui` in this case, we can also use relative path.

```suggestion
    <Link href="/" external={false}>
```

Reply:
That's a much cleaner way of doing it! Shows how inexperienced I am with React as of now haha. Putting up a new PR, thanks!